packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -396,6 +394,12 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         resolvedAt = Timestamp.wrap(uint64(block.timestamp));
 
         emit Resolved(status = status_);
+
+        // If the game resolved as `DEFENDER_WINS`, the root claim was deemed correct. Attempt to set the
+        // latest finalized output root.
+        if (status_ == GameStatus.DEFENDER_WINS) {
+            PORTAL.trySetLatestFinalizedOutputRoot();
```

Comment:
note: this doesn't respect the finalization delay for resolved games in the portal. This value is only used to set up the initial anchor in the games, and it's nice to have it set along with the global resolution, but we probably need to make this respect the delay and have an extra poke.